test/responses_unit_test.go

```diff
@@ -51,7 +50,7 @@ func weMakeAnyCallTo(client /* algod/indexer */, endpoint string) (err error) {
 		switch endpoint {
 		case "lookupAccountByID":
 			round, something, err = indexerC.LookupAccountByID("").Do(context.Background())
-			response = models.LookupAccountByIDResponse{
+			response = models.AccountResponse{
```

Comment:
In order for the "decomposed" versions of the API to be compared against the actual results, the full response object must be reconstructed.

These reconstructed objects are outside the scope of the client interface, so it doesn't matter that the names have changed.

I also added some stuff that was missing from adding the full json-compare to the other 2 test file changes.